addons/payment_sips/__manifest__.py

```diff
@@ -2,16 +2,12 @@
 # Original Copyright 2015 Eezee-It, modified and maintained by Odoo.
 
 {
-    'name': 'Worldline SIPS',
+    'name': 'Payment Provider: Worldline SIPS',
     'version': '2.0',
     'category': 'Accounting/Payment Providers',
-    'sequence': 390,
-    'description': """
-Worldline SIPS Payment Provider for online payments
-
-Implements the Worldline SIPS API for payment providers.
-Other SIPS providers may be compatible, though this is
-not guaranteed.""",
+    'sequence': 350,
+    'summary': "A Wordline payment provider for online payments supporting several payment methods "
```

Comment:
`Worldline` is the name of the company, since it's already said in the name, I believe we can remove it from the summary to be more consistent regarding the other module summaries.